tests/unit/mock/config/parser/nokia_sros/sros_full_sent.txt

```diff
@@ -0,0 +1,256 @@
+configure { log filter "1001" named-entry "10" action forward }
+configure { log filter "1001" named-entry "10" match }
+configure { log filter "1001" named-entry "10" match severity }
+configure { log filter "1001" named-entry "10" match severity gte major }
+configure { log log-id "100" }
+configure { log log-id "100" description "Default Serious Errors Log" }
+configure { log log-id "100" filter "1001" }
+configure { log log-id "100" source }
+configure { log log-id "100" source main true }
+configure { log log-id "100" destination }
+configure { log log-id "100" destination memory }
+configure { log log-id "100" destination memory max-entries 500 }
+configure { log log-id "99" }
+configure { log log-id "99" description "Default System Log" }
+configure { log log-id "99" source }
+configure { log log-id "99" source main true }
+configure { log log-id "99" destination }
+configure { log log-id "99" destination memory }
+configure { log log-id "99" destination memory max-entries 500 }
+configure { port 1/1/c1 }
+configure { port 1/1/c2 }
+configure { port 1/1/c3 }
+configure { port 1/1/c4 }
+configure { port 1/1/c5 }
+configure { port 1/1/c6 }
+configure { port 1/1/c7 }
+configure { port 1/1/c8 }
+configure { port 1/1/c9 }
+configure { port 1/1/c10 }
+configure { port 1/1/c11 }
+configure { port 1/1/c12 }
+configure { router "Base" }
+configure { router "Base" interface "L3-OAM-eNodeB069420-W1" }
+configure { router "Base" interface "L3-OAM-eNodeB069420-W1" admin-state disable }
+configure { router "Base" interface "L3-OAM-eNodeB069420-W1" ingress-stats false }
+configure { system }
+configure { system name "core-router" }
+configure { system grpc }
+configure { system grpc admin-state enable }
+configure { system grpc allow-unsecure-connection }
+configure { system grpc gnmi }
+configure { system grpc gnmi auto-config-save true }
+configure { system grpc rib-api }
+configure { system grpc rib-api admin-state enable }
+configure { system management-interface }
+configure { system management-interface configuration-mode model-driven }
+configure { system management-interface netconf }
+configure { system management-interface netconf admin-state enable }
+configure { system management-interface netconf auto-config-save true }
+configure { system management-interface snmp }
+configure { system management-interface snmp packet-size 9216 }
+configure { system management-interface snmp streaming }
+configure { system management-interface snmp streaming admin-state enable }
+configure { system bluetooth }
+configure { system bluetooth advertising-timeout 30 }
+configure { system login-control }
+configure { system login-control ssh }
+configure { system login-control ssh inbound-max-sessions 30 }
+configure { system security }
+configure { system security aaa }
+configure { system security aaa local-profiles }
+configure { system security aaa local-profiles profile "administrative" }
```

Comment:
Why are things like this quoted? Is there potential for a space, and would that effect the parsing?

Reply:
This appears to be the formating of SrOS and from what I've seen, it's always a single space around the curly brackets. I believe this is a collapsed, single-line form of the tab indented format that doesn't include the repeated configuration lines, ie the `configuration system security aaa` lines in this example.